Authenticators/LDAP/LDAPauth.py

```diff
@@ -97,18 +98,24 @@
 # authentication scheme.
 #
 #    Requirements:
-#        * python >=2.4 and the following python modules:
+#        * python >=3.8 (maybe 3.6 is enought but it wasn't tested) and the following python modules:
 #            * ice-python
 #            * python-ldap
 #            * daemon (when run as a daemon)
+#    If you are using Ubuntu/Debian (only Ubuntu 20.04 was tested) this means the following packages:
+#        * python3
+#        * python3-zeroc-ice
+#        * python3-ldap
+#        * python3-daemon
+#        * zeroc-ice-slice
 
 import sys
 import ldap
 import Ice
-import thread
-import urllib2
+import _thread
```

Comment:
What’s the significance of the underscore prefix? What does it mean?

Reply:
It's the module name: https://docs.python.org/3/library/_thread.html

Reply:
But why do they add an underscore to it?

Reply:
Thats a great question to ask the python developer why they name their module like that but not related to this pull request... the python 2 "thread" module is simply named "_thread" in python 3